core/finality-grandpa/src/communication/gossip.rs

```diff
@@ -94,6 +95,15 @@ enum Consider {
 	RejectPast,
 	/// Message is from the future. Reject.
 	RejectFuture,
+    /// Message is an equivocation. Reject.
+    RejectEquivocation,
```

Comment:
spaces

Reply:
I think this should also be `RejectRedundant`, because we do accept _some_ equivocations.